jobs/service-fabrik-broker/spec

```diff
@@ -73,6 +76,7 @@ provides:
   - backup.abort_time_out
   - backup.consecutive_backup_failure_sla_count
   - backup.provider
+  - ha.provider
```

Comment:
the attrib ha.provider having the value landscape type is misleading. To me HA Provider here in this case should be sth like 'keepalived' or 'loadbalancer' etc.

Reply:
will rename to 'ha.iaas_config' and this has been moved to keepalived job spec